common/source/docs/common-oem-customizations.rst

```diff
@@ -51,4 +51,9 @@ This assumes that the OEM has setup the build environment (:ref:`building-the-co
 
 6. Now build as normal with OEM-MatekF405-Wing as the board name in the configuration. The defaults and scripts will be embedded appropriately.
 
+Alternative To Customizing hwdef.dat
+------------------------------------
+
+Instead of creating a separate branch and modifying the hwdef file, you can also insert LUA scripts or even informational files into the ROMFS of the build, by simply going to the local ardupilot reo's "build" folder and creating a sub-folder named "ROMFS_custom". Place your LUA scripts in a subfolder of this directory, named "scripts". You can have other suboflders for informational files and these will be included and viewable when examining the @SYS folder with MAVftp.
```

Comment:
spelling reo's -> repositories
spelling suboflders -> subfolders
the files will be viewable in the @ROMFS folder with MAVFtp, not the @SYS folder